recipes/blast/build.sh

```diff
@@ -42,11 +39,8 @@ LIB_INSTALL_DIR=$PREFIX/lib/ncbi-blast+
 # -vdb: disable VDB/SRA toolkit
 # z: set zlib
 # bz2: set libbz2
-# boost: set boost
 # -openssl: disable openssl
 # -gcrypt: disable gcrypt (needed on OSX)
-# gnutls: set gnutls (preferred over openssl for -remote support)
-# nettle: set nettle
```

Comment:
Do we still have support for `-remote` then?

Reply:
Yes.